[2023-09-11 11:56:55,144] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: listing_4_20.extract_gz scheduled__2023-09-10T16:00:00+00:00 [queued]>
[2023-09-11 11:56:55,168] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: listing_4_20.extract_gz scheduled__2023-09-10T16:00:00+00:00 [queued]>
[2023-09-11 11:56:55,168] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-09-11 11:56:55,169] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-09-11 11:56:55,169] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-09-11 11:56:55,204] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): extract_gz> on 2023-09-10 16:00:00+00:00
[2023-09-11 11:56:55,213] {standard_task_runner.py:52} INFO - Started process 962 to run task
[2023-09-11 11:56:55,219] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'listing_4_20', 'extract_gz', 'scheduled__2023-09-10T16:00:00+00:00', '--job-id', '433', '--raw', '--subdir', 'DAGS_FOLDER/listing_4_20.py', '--cfg-path', '/tmp/tmpxoiidcp7', '--error-file', '/tmp/tmpyrwxojgt']
[2023-09-11 11:56:55,220] {standard_task_runner.py:80} INFO - Job 433: Subtask extract_gz
[2023-09-11 11:56:55,354] {task_command.py:371} INFO - Running <TaskInstance: listing_4_20.extract_gz scheduled__2023-09-10T16:00:00+00:00 [running]> on host b18943745c0d
[2023-09-11 11:56:55,496] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=listing_4_20
AIRFLOW_CTX_TASK_ID=extract_gz
AIRFLOW_CTX_EXECUTION_DATE=2023-09-10T16:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-09-10T16:00:00+00:00
[2023-09-11 11:56:55,498] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-09-11 11:56:55,500] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'gunzip --force /tmp/wikipageviews.gz']
[2023-09-11 11:56:55,517] {subprocess.py:85} INFO - Output:
[2023-09-11 11:56:59,963] {subprocess.py:92} INFO - 
[2023-09-11 11:56:59,964] {subprocess.py:92} INFO - gzip: /tmp/wikipageviews.gz: decompression OK, trailing garbage ignored
[2023-09-11 11:57:00,015] {subprocess.py:92} INFO - gzip: /tmp/wikipageviews.gz: No such file or directory
[2023-09-11 11:57:00,016] {subprocess.py:96} INFO - Command exited with return code 2
[2023-09-11 11:57:00,049] {taskinstance.py:1909} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 195, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-09-11 11:57:00,061] {taskinstance.py:1420} INFO - Marking task as FAILED. dag_id=listing_4_20, task_id=extract_gz, execution_date=20230910T160000, start_date=20230911T115655, end_date=20230911T115700
[2023-09-11 11:57:00,085] {standard_task_runner.py:97} ERROR - Failed to execute job 433 for task extract_gz (Bash command failed. The command returned a non-zero exit code 2.; 962)
[2023-09-11 11:57:00,152] {local_task_job.py:156} INFO - Task exited with return code 1
[2023-09-11 11:57:00,228] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
